dist/tokens/scss/typography.scss

```diff
@@ -1,126 +1,22 @@
-$h1FontWeight: 700;
-$h1LineHeight: 40px;
-$h1FontSize: 40px;
-$h1LetterSpacing: 0;
-$h1ParagraphSpacing: 0;
-$h1TextDecoration: none;
-$h1TextCase: none;
-$h2FontFamily: Mulish;
-$h2FontWeight: 700;
-$h2LineHeight: 40px;
-$h2FontSize: 32px;
-$h2LetterSpacing: 0;
-$h2ParagraphSpacing: 0;
-$h2TextDecoration: none;
-$h2TextCase: none;
-$h3FontFamily: Mulish;
-$h3FontWeight: 700;
-$h3LineHeight: 32px;
-$h3FontSize: 24px;
-$h3LetterSpacing: 0;
-$h3ParagraphSpacing: 0;
-$h3TextDecoration: none;
-$h3TextCase: none;
-$h4FontFamily: Mulish;
-$h4FontWeight: 700;
-$h4LineHeight: 24px;
-$h4FontSize: 20px;
-$h4LetterSpacing: 0;
-$h4ParagraphSpacing: 0;
-$h4TextDecoration: none;
-$h4TextCase: none;
-$h5FontFamily: Mulish;
-$h5FontWeight: 700;
-$h5LineHeight: 24px;
-$h5FontSize: 16px;
-$h5LetterSpacing: 0;
-$h5ParagraphSpacing: 0;
-$h5TextDecoration: none;
-$h5TextCase: none;
-$bodyFontFamily: Mulish;
-$bodyFontWeight: 500;
-$bodyLineHeight: 24px;
-$bodyFontSize: 16px;
-$bodyLetterSpacing: 0;
-$bodyParagraphSpacing: 0;
-$bodyTextDecoration: none;
-$bodyTextCase: none;
-$footnoteFontFamily: Mulish;
-$footnoteFontWeight: 500;
-$footnoteLineHeight: 16px;
-$footnoteFontSize: 14px;
-$footnoteLetterSpacing: 0;
-$footnoteParagraphSpacing: 0;
-$footnoteTextDecoration: none;
-$footnoteTextCase: none;
-$captionFontFamily: Mulish;
-$captionFontWeight: 500;
-$captionLineHeight: 16px;
-$captionFontSize: 12px;
-$captionLetterSpacing: 0;
-$captionParagraphSpacing: 0;
-$captionTextDecoration: none;
-$captionTextCase: none;
-$subcaptionFontFamily: Mulish;
-$subcaptionFontWeight: 500;
-$subcaptionLineHeight: 16px;
-$subcaptionFontSize: 10px;
-$subcaptionLetterSpacing: 0;
-$subcaptionParagraphSpacing: 0;
-$subcaptionTextDecoration: none;
-$subcaptionTextCase: none;
-$mobileH1FontFamily: Mulish;
-$mobileH1FontWeight: 700;
-$mobileH1LineHeight: 40px;
-$mobileH1FontSize: 32px;
-$mobileH1LetterSpacing: 0;
-$mobileH1ParagraphSpacing: 0;
-$mobileH1TextDecoration: none;
-$mobileH1TextCase: none;
-$mobileH2FontFamily: Mulish;
-$mobileH2FontWeight: 700;
-$mobileH2LineHeight: 32px;
-$mobileH2FontSize: 28px;
-$mobileH2LetterSpacing: 0;
-$mobileH2ParagraphSpacing: 0;
-$mobileH2TextDecoration: none;
-$mobileH2TextCase: none;
-$mobileH3FontFamily: Mulish;
-$mobileH3FontWeight: 700;
-$mobileH3LineHeight: 32px;
-$mobileH3FontSize: 22px;
-$mobileH3LetterSpacing: 0;
-$mobileH3ParagraphSpacing: 0;
-$mobileH3TextDecoration: none;
-$mobileH3TextCase: none;
\ No newline at end of file
+$h1: [object Object];
+$h2: [object Object];
+$h3: [object Object];
+$h4: [object Object];
+$h5: [object Object];
+$body: [object Object];
+$footnote: [object Object];
+$caption: [object Object];
+$subcaption: [object Object];
+$mobileH1: [object Object];
+$mobileH2: [object Object];
+$mobileH3: [object Object];
+$mobileH4: [object Object];
+$mobileH5: [object Object];
+$mobileBody: [object Object];
+$mobileFootnote: [object Object];
+$mobileCaption: [object Object];
+$mobileSubcaption: [object Object];
```

Comment:
this isn't being used anywhere so we _could_ let slide for now, but we'll have to fix this at some point. or yeet typography altogether for scss until we need it

Reply:
Oh lordy 😂 Yeah I think we can probably yeet this and convert to themes. I just asked Mirre, and they ain't using them. I could just remove it from the build